<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuctionsWidget</class>
 <widget class="QWidget" name="FuctionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>460</height>
    </rect>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>460</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="right" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>771</width>
      <height>461</height>
     </rect>
    </property>
    <widget class="QWidget" name="scrollview" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>771</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>771</width>
       <height>61</height>
      </rect>
     </property>
     <widget class="QLabel" name="redian">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>热点</string>
      </property>
     </widget>
     <widget class="QLabel" name="question1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>显卡,声卡出故障?</string>
      </property>
     </widget>
     <widget class="QWidget" name="line" native="true">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>15</y>
        <width>1</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="line_2" native="true">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>15</y>
        <width>1</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>浏览器主页被劫持?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>上不了网,怎么办?</string>
      </property>
     </widget>
     <widget class="QWidget" name="search" native="true">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>5</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="searchIcon">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>5</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="listWidgetBackGround" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>460</height>
     </rect>
    </property>
   </widget>
   <zorder>listWidgetBackGround</zorder>
   <zorder>listWidget</zorder>
   <zorder>right</zorder>
  </widget>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>460</height>
    </rect>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>main</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
